### PR TITLE
Show self-hosted url in example introduction

### PR DESCRIPTION
**Pull request by JoshuaBehrens**

### CODE REVIEW

File: templates/account.html.twig [hunk 1]
```diff
@@ -36,7 +36,7 @@
 "repositories": [
     {
         "type": "composer",
-        "url": "https://packages.friendsofshopware.com/",
+        "url": "{{ url('home') }}",
```
Comment (shyim) [inline templates/account.html.twig:39]: Can you use APP_URL the env?

Comment (JoshuaBehrens) [reply templates/account.html.twig:39]: Isn't APP_URL affecting how the home route is structured?

Comment (shyim) [reply templates/account.html.twig:39]: Nope

Comment (JoshuaBehrens) [reply templates/account.html.twig:39]: But it works out for me like this :( but I am fine with using the env as well. Do I have to make sure it has a trailing slash?

**Pull request merged**